pilot/pkg/proxy/envoy/v2/eds.go

```diff
@@ -248,20 +248,20 @@ func (s *DiscoveryServer) updateServiceShards(push *model.PushContext) error {
 
 	// TODO: if ServiceDiscovery is aggregate, and all members support direct, use
```

Comment:
Nit; I think the TODO might need to either go or get updated since we no longer have the aggregate Service Discovery.